docs/references.rst

```diff
@@ -53,3 +53,10 @@
 	     Austin, TX, 2019. doi:`10.25080/Majora-7ddc1dd1-013`_.
 
 .. _`10.25080/Majora-7ddc1dd1-013` : https://doi.org/10.25080/Majora-7ddc1dd1-013
+
+.. [Awtrey2019] Awtrey, Nikolaus. SPIDAL Summer REU 2019: Parallelizing
+                DensityAnalysis and RMSF in PMDA. Technical report, Arizona
+                State University, Tempe, AZ, 2019.
+                doi: `10.6084/m9.figshare.9695852.v1`_. 
+
+.. _`10.6084/m9.figshare.9695852.v1` : https://figshare.com/articles/SPIDAL_Summer_REU_2019_Parallelizing_DensityAnalysis_and_RMSF_in_PMDA/9695852
```

Comment:
WHen using DOI it is better to use the DOI resolver as that one is guaranteed to always resolve to the actual paper
```reST
.. _`10.6084/m9.figshare.9695852` : https://doi.org/10.6084/m9.figshare.9695852
```

ALso better to cite the top doi and not a specific version.